xbmc/filesystem/test/TestDirectoryCache.cpp

```diff
@@ -0,0 +1,514 @@
+/*
+ *  Copyright (C) 2017-2018 Team Kodi
```

Comment:
Can you do `Copyright (C) 2025`? All code is written in a context, and the year is relevant to its context, so I like new files to be the latest year to help readers know the file's context.